packages/block-library/src/navigation-link/edit.js

```diff
@@ -133,6 +133,31 @@ function getSuggestionsQuery( type, kind ) {
 	}
 }
 
+/**
+ * Link control may sometimes return a type which we do not want to persist in block attributes.
+ *
+ * @param {string} type The type returned by the link control suggestion.
+ * @return {{}|{type: string}} empty object or object with type of string
```

Comment:
```suggestion
 * @return {{}|{type: string}} object with the correctly mapped `type` property to be set on the `block.attribute.type` based. 
```

It feels like we're currently describing what is already documented in the JSDoc `@param` syntax.

I wonder whether we should amend this to be more descriptive of what we are getting back.